pkg/reconciler/v1alpha1/processor/resources/deployment.go

```diff
@@ -91,3 +75,36 @@ func MakeDeployment(proc *streamv1alpha1.Processor) (*appsv1.Deployment, error)
 
 	return deployment, nil
 }
+
+func computeEnvironmentVariables(processor *streamv1alpha1.Processor) []corev1.EnvVar {
+	return append([]corev1.EnvVar{
+		{
+			Name:  "INPUTS",
+			Value: strings.Join(processor.Status.InputAddresses, ","),
+		},
+		{
+			Name:  "OUTPUTS",
+			Value: strings.Join(processor.Status.OutputAddresses, ","),
+		},
+		{
+			Name:  "GROUP",
+			Value: processor.Name,
+		},
+		{
+			Name:  "FUNCTION",
+			Value: "localhost:8080",
+		},
+	}, exposeOutputContentTypes("OUTPUT_CONTENT_", processor.Status.OutputContentTypes)...)
```

Comment:
No need to parameterize the prefix, in the same way that INPUTS, OUTPUTS, etc is hardcoded (or could become a const) here

Reply:
it's just to co-locate them all, it serves no other purposes.

Reply:
perhaps a higher level config format should be defined

Reply:
Could do, although this is entirely private to riff and never exposed to the user

Reply:
> Could do, although this is entirely private to riff and never exposed to the user

All the more reason we can pass a blob of json or some other format guaranteed to never have encoding issues.

Reply:
done now